infra/src/main/java/com/redhat/service/smartevents/infra/models/dto/ProcessorDTO.java

```diff
@@ -107,6 +110,26 @@ public void setKafkaConnection(KafkaConnectionDTO kafkaConnection) {
         this.kafkaConnection = kafkaConnection;
     }
 
+    public Optional<KafkaConnectionDTO> parseKafkaOutgoingConnection() {
+        Action action = definition.getResolvedAction();
+        String brokerUrl = action.getParameter(KafkaTopicConstants.KAFKA_BROKER_URL);
+        String clientId = action.getParameter(KafkaTopicConstants.KAFKA_CLIENT_ID);
+        String clientSecret = action.getParameter(KafkaTopicConstants.KAFKA_CLIENT_SECRET);
+        String topic = action.getParameter(KafkaTopicConstants.TOPIC_PARAM);
+        String securityProtocol = action.getParameter(KafkaTopicConstants.KAFKA_SECURITY_PROTOCOL);
+
+        if (brokerUrl != null
```

Comment:
Maybe it would be good to provide some info using logs etc. about what is missing. There are many parameters which need to be set.

Reply:
I've updated the ACTIONS.md file as well with the instructions